test/FortranArray.ll

```diff
@@ -10,7 +10,7 @@
 ; and back to .ll. This causes the LLVM IR verifier to fail as there
 ; are different rules for valid DISubRange depending on language ID.
 
-; CHECK-LLVM: !DISubrange(lowerBound: 1)
+; CHECK-LLVM: !DISubrange(lowerBound: 1, upperBound: !DIExpression(DW_OP_push_object_address, DW_OP_plus_uconst, 64, DW_OP_deref, DW_OP_push_object_address, DW_OP_plus_uconst, 48, DW_OP_deref, DW_OP_plus, DW_OP_constu, 1, DW_OP_minus))
```

Comment:
Can we preserve the expression for the lower bound too?

Reply:
Now - yes: 8d7e19c